docs/team/index.md

```diff
@@ -22,7 +22,7 @@ conversations about extensibility of Core Jekyll.*
 1. Ashwin (@ashmaroli)
 2. Florian (@Crunch09)
 3. Mert (@mertkahyaoglu, jekyll-admin)
-4. Chase (@Chase
+4. Alfred Xing (@alfredxing)
```

Comment:
@Ch4s3 is still maintaining `classifier-reborn` whether @alfredxing is not maintaining any plugin at the moment.

Reply:
I'm not yet maintaining any of the *Official Plugins* as well..

Reply:
Ahh, yes, I am still maintaining `classifier-reborn`.